sw/airborne/arch/chibios/mcu_periph/i2c_arch.c

```diff
@@ -86,38 +86,38 @@ static void handle_i2c_thd(struct i2c_periph *p)
   if (t->len_w > 0) {
 #if defined STM32F7
     // we do stupid mem copy because F7 needs a special RAM for DMA operation
-    memcpy(i->dma_buf, (void*)t->buf, (size_t)(t->len_w));
+    memcpy(i->dma_buf, (void *)t->buf, (size_t)(t->len_w));
     status = i2cMasterTransmitTimeout(
-        (I2CDriver*)p->reg_addr,
-        (i2caddr_t)((t->slave_addr)>>1),
-        (uint8_t*)i->dma_buf, (size_t)(t->len_w),
-        (uint8_t*)i->dma_buf, (size_t)(t->len_r),
-        tmo);
-    memcpy((void*)t->buf, i->dma_buf, (size_t)(t->len_r));
+               (I2CDriver *)p->reg_addr,
```

Comment:
stylecheck pls?

Reply:
Fix_code_style is actually what caused this...

Reply:
I wrote this part of the code. The code_style script should be correct, it is probably my VIM settings for aligning multi-line parameters that are a bit different.